changelog.md

```diff
@@ -2,6 +2,8 @@
 
 ## Not yet released (VerneMQ 1.4.0)
 
+- Add `xmerl` and `inets` from the Erlang standard library to the release in order
+  to allow plugin developers to have these libraries available.
```

Comment:
Could you move this to the bottom of the list (rebase first, we just merged a fix for the port-compiler version issue).

going forward we'll always try to add things at the bottom of the list, it makes it easier to know what was added when :)

Reply:
Nice, i will, didn't read your entire message above

Reply:
Done!